src/main/java/domain/Prize.java

```diff
@@ -0,0 +1,68 @@
+package domain;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Prize {
+
+    private static final String INVALID_PRIZE_LENGTH_ERROR_MESSAGE = "사다리 게임의 실행 결과는 1글자에서 5글자 사이로 입력해야합니다.";
+    private static final String INVALID_PRIZE_COUNT_ERROR_MESSAGE = "사다리 게임의 실행 결과는 사람 수와 동일하게 입력해야합니다.";
+    private static final int PRIZE_LENGTH_LOWER_BOUND = 1;
+    private static final int PRIZE_LENGTH_UPPER_BOUND = 5;
+
+
+    private final List<String> prizes;
+
+    public Prize(List<String> prizes, int personCount) {
+        List<String> timedPrizes = trimPrizes(prizes);
+        validate(timedPrizes, personCount);
+        this.prizes = timedPrizes;
+    }
+
+    private void validate(List<String> prizes, int personCount) {
+        validatePrizesCount(prizes, personCount);
+        validatePrizesLength(prizes);
+    }
+
+    private void validatePrizesCount(List<String> prizes, int personCount) {
+        if (!isSameCount(prizes, personCount)) {
+            throw new IllegalArgumentException(INVALID_PRIZE_COUNT_ERROR_MESSAGE);
+        }
+    }
+
+    private boolean isSameCount(List<String> prizes, int personCount) {
+        return prizes.size() == personCount;
+    }
+
+    private void validatePrizesLength(List<String> prizes) {
+        if (hasInvalidPrizeLength(prizes)) {
+            throw new IllegalArgumentException(INVALID_PRIZE_LENGTH_ERROR_MESSAGE);
+        }
+    }
+
+    private boolean hasInvalidPrizeLength(List<String> prizes) {
+        return prizes.stream()
+                .filter(prize -> isInvalidNameLength(prize))
+                .count() != 0;
```

Comment:
```suggestion
return prizes.stream().anyMatch(prize -> isInvalidNameLength(prize));
```

같은 역할을 하는 메서드가 있습니다-! :)

Reply:
오호...!
새롭게 알아갑니다 ㅎㅎ

stream 내에서 쓰는 것만 쓰던 습관이 있어서 결과에만 초점을 맞췄었는데
앞으로는 조금 더 기능별 목적에 맞게 사용하도록 노력해보겠습니다 ㅎㅎ
감사합니다!